rosetta-runtime/src/main/java/com/rosetta/model/lib/validation/ValidationData.java

```diff
@@ -0,0 +1,4 @@
+package com.rosetta.model.lib.validation;
+
+public class ValidationData {
```

Comment:
This class should be abstract.

Reply:
if we change this it won't work for this 
![image](https://github.com/REGnosys/rosetta-dsl/assets/107400406/fdb90c9f-7509-4c0a-b855-92052568bfff)

Reply:
Agreed. I think we should change it, and modify this method. See comment below:

> I think we should deprecate this one too, and rather create a `KeyValidator implements RosettaModelObjectValidator<Key>` using our new infrastructure.